templates/zerver/help/view-messages-from-a-user.md

```diff
@@ -7,8 +7,8 @@ the following steps.
     **Note**: Your private message history will not include messages sent in
     group private messages.
 
-1. Click on the name of the user that you want to see your messages with in the
-user's list located at the right sidebar.
+1. Click on the name of the user that you want to see your messages with, in the
+user's list located on the right sidebar.
```

Comment:
This would probably be more clearly phrased as "In the right sidebar user list, click the name of the user."